.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,13 +1,13 @@
 ---
 name: Bug report
-about: Create a report to help us improve Project Smoothie
+about: Create a report to help us improve the Open Food Facts mobile app (iPhone and Android)
 title: ""
-labels: ""
+labels: "bug"
 assignees: ""
 ---
 
 ### What
-- Describe the issue clearly
+- <!-- Describe the issue clearly -->
```

Comment:
"xml commenting" all the comments will make them invisible for who writes a new issue, right? Not convinced it's a good idea. But why not?

Reply:
It will be visible in the fill form, I believe, let's check